Anton_Bartoshevich/08.Docker.Docker-compose/docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
```

Comment:
according to the task, you need to use tag as initiator of deployment

Reply:
OK, I'll repare it